src/torchcodec/samplers/__init__.py

```diff
@@ -0,0 +1 @@
+from ._implem import clips_at_random_indices
```

Comment:
I'm creating a new `torchcodec.samplers` namespace, keeping the existing `torchcodec._samplers` namespace untouched to avoid conflicts on internal codebase.